src/algebra/field.lean

```diff
@@ -1,45 +1,222 @@
 /-
-Copyright (c) 2017 Johannes Hölzl. All rights reserved.
+Copyright (c) 2014 Robert Lewis. All rights reserved.
```

Comment:
This copyright date does not seem to be accurate.

Reply:
I think most mathlib files contain the date of the first revision. The name and date here is from the `init.algebra.field` file. If we want to change this to `(c) 2014-2020` or something then that should be applied to a lot more theorems than this.